.changeset/serious-crabs-begin.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+chore: make `build:schema` script self-sufficient
```

Comment:
Is there any docs about this that we should update?

Reply:
No, this is an internal utility we use to have our `fuel-core-schema.graphql` file always in sync with the latest `fuel-core` version the sdk uses.